docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,141 @@
 
-#. Deactivate the snakes environment and return to base environment:
-   ``conda activate``
+Specifying channels
+===================
 
-   .. note::
-      For versions prior to conda 4.6, use:
+Channels are locations (on your own computer or elsewhere on the Internet) where packages are stored. By default, conda searches for packages in its :ref:`default channels <default-channels>`.
 
-        * Windows:  ``activate``
-        * macOS, Linux: ``source activate``
+If a package you want is located in another channel, such as conda-forge, you can manually specify the channel when installing the package::
 
+   conda activate myenvironment
+   conda install conda-forge::numpy
 
-.. _managing-pkgs:
+You can also override the default channels in your `.condarc` file. For a direct example, see :ref:`Channel locations (channels) <config-channels>` or read the entire :doc:`Using the .condarc conda configuration file <configuration/use-condarc>`.
 
-Managing packages
-=================
+.. tip::
 
-In this section, you check which packages you have installed,
-check which are available and look for a specific package and
-install it.
+   Find more packages and channels by searching `Anaconda.org <https://www.anaconda.org>`_. To learn more about searching for packages and channels on Anaconda.org, see `Anaconda.org's documentation <https://docs.anaconda.com/free/anacondaorg/user-guide/>`_.
 
-#. To find a package you have already installed, first activate the environment
-   you want to search. Look above for the commands to
-   :ref:`activate your snakes environment <managing-envs>`.
+Updating conda
+==============
 
-#. Check to see if a package you have not installed named
-   "beautifulsoup4" is available from the Anaconda repository
-   (must be connected to the Internet):
+To see your conda version, use the following command::
 
-   .. code::
+   conda --version
 
-      conda search beautifulsoup4
+No matter which environment you run this command in, conda displays its current version::
 
-   Conda displays a list of all packages with that name on the Anaconda
-   repository, so we know it is available.
+   conda 23.10.0
 
-#. Install this package into the current environment:
+.. note::
+   If you get an error message ``command not found: conda``, close and reopen
+   your terminal window and verify that you are logged
+   into the same user account that you used to install conda.
 
-   .. code::
+To update conda to the latest version::
 
-      conda install beautifulsoup4
+   conda activate
+   conda update conda
 
-#. Check to see if the newly installed program is in this environment:
+Conda compares your version to the latest available version and then displays what is available to install.
 
-   .. code::
+If a newer version of conda is available, type ``y`` and press Enter to update::
 
-      conda list
+   Proceed ([y]/n)? y
 
+.. tip::
+   We recommend that you always keep conda updated to the latest version.
```

Comment:
Opportunity to point to [CEP 10](https://github.com/conda-incubator/ceps/blob/main/cep-10.md)?

Reply:
Sure!